### PR TITLE
adding flatpak manifest + patch to create flatpak application

### PR DESCRIPTION
**Pull request by jeteokeeffe**
Adding flatpak manifest which requests a patch to look for libs, includes in /app.

### CODE REVIEW

**Comment by Calinou** [comment]
Shouldn't the manifest be submitted on [Flathub](https://github.com/flathub/flathub) as a pull request?

**Comment by jeteokeeffe** [comment]
@Calinou Yes, it should (https://github.com/flathub/flathub/wiki/App-Submission#how-to-submit-an-app ). You should also have it in your own project's repo.

File: org.dust3d.dust3d.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                "cd thirdparty/instant-meshes/build && make -j 4",
+                "qmake -config release",
+                "make CXX=\"g++ -fext-numeric-literals\"",
+                "install -D dust3d /app/bin/dust3d"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/huxingyi/dust3d.git",
+                    "tag": "1.0.0-rc.3"
```
Comment (huxingyi) [inline org.dust3d.dust3d.json:69]: Could you try 1.0.0-rc.6 please?

**Comment by jeteokeeffe** [comment]
Sure. I've been building with QT 5.13 but should it be 5.9?
    On Wednesday, April 15, 2020, 5:54:17 PM EDT, Jeremy HU <notifications@github.com> wrote:  
 
 


@huxingyi commented on this pull request.

In org.dust3d.dust3d.json:
> +        {
+            "name": "dust3d",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p thirdparty/instant-meshes/build",
+                "cd thirdparty/instant-meshes/build && cmake ../",
+                "cd thirdparty/instant-meshes/build && make -j 4",
+                "qmake -config release",
+                "make CXX=\"g++ -fext-numeric-literals\"",
+                "install -D dust3d /app/bin/dust3d"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/huxingyi/dust3d.git",
+                    "tag": "1.0.0-rc.3"

Could you try 1.0.0-rc.6 please?

**Comment by huxingyi** [comment]
5.13 is better as long as it doesn’t require user to manually install some dependencies. I am not familiar with flatpak, you decide.

**Comment by jeteokeeffe** [comment]
Did some basic testing and commited rc 6

**Pull request merged**